### PR TITLE
Getting constants changed

### PR DESCRIPTION
**Pull request by NeimadTL**
I did  the rebase right here with @kotp. Everything should be clean.

### CODE REVIEW

**Comment by Insti** [comment]
There are still a lot of strangely named files included in these commits. 
I suggest checking out a fresh branch and manually re-applying the changes you want to make.

**Comment by kotp** [comment]
a18e633 is the commit bringing these in.  However, look deeper as to what caused the files to happen.

**Comment by NeimadTL** [comment]
I regenated all the tests and there is no difference

File: docs/24pullrequests.md [hunk 1]
```diff
@@ -18,4 +18,4 @@ Something missing?  Create a pull request!
 
 Check out the [Getting Involved in a Track](https://github.com/exercism/exercism.io/blob/master/docs/getting-involved-in-a-track.md) documentation, and some [ideas for things we are looking for](http://exercism.io/languages/ruby/todo) for exercises that are needed to be implemented.
 
-We are available to answer questions in the  [![Join the chat at https://gitter.im/exercism/ruby](https://badges.gitter.im/exercism/ruby.svg)](https://gitter.im/exercism/ruby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge).
+We are available to answer questions in the  [![Join the chat at https://gitter.im/exercism/ruby](https://badges.gitter.im/exercism/ruby.svg)](https://gitter.im/exercism/ruby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge).
```
Comment (Insti) [inline docs/24pullrequests.md]: I can't tell what the change is here.

Comment (kotp) [reply docs/24pullrequests.md]: I can't either, but the two spaces after the word "the" could be fixed.  It is stating that there is a "non-visible" change in that line.  Hex Editor... here we come!

Comment (kotp) [reply docs/24pullrequests.md]: It could be a line-ending change, that ends up not being visible, yet still a change.

**Comment by Insti** [comment]
Thanks @NeimadTL

**Comment by NeimadTL** [comment]
you're welcome.

**Pull request merged**